### PR TITLE
update the ZMK firmware descritpion

### PR DESCRIPTION
**Pull request by 280Zo**
The scope of this PR is to update some of the wording about how to use the firmware. It also includes a short description about the firmware.
I've updated the firmware repo that's linked in this doc and it now builds both qwerty and colemak on the main branch. So it might be a good idea to update the wording to keep from confusing users who click on the link looking for different branches.
Cheers!

### CODE REVIEW

File: help/bluetooth.md [hunk 1]
```diff
@@ -179,17 +179,14 @@ You can either mount the battery between the MCU and holder PCB, or tape it to t
 
 # Firmware
 
-The firmware can be downloaded from the [charybdis-wireless-mini-zmk-firmware repository](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware) by opening [the Actions tab](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware/actions), clicking on the latest successful run, then downloading the firmware file under Artifacts.
+The firmware can be downloaded from the [charybdis-wireless-mini-zmk-firmware](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware) repository by opening the [Actions Workflows](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware/actions?query=is%3Acompleted+branch%3Amain), selecting latest successful run on the main branch, then downloading the `firmware-charybdis-nanov2-qwerty` artifacr. There's also a colemak dh firmware for those that want to use that layout. Note that you have to be logged into Github to be able to download the files.
```
Comment (burkfers) [inline help/bluetooth.md]:
```suggestion
The firmware can be downloaded from the [charybdis-wireless-mini-zmk-firmware](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware) repository by opening the [Actions Workflows](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware/actions?query=is%3Acompleted+branch%3Amain), selecting latest successful run on the main branch, then downloading the `firmware-charybdis-nanov2-qwerty` artifact. There's also a colemak dh firmware for those that want to use that layout. Note that you have to be logged into Github to be able to download the files.
```
Typo fix

File: help/bluetooth.md [hunk 2]
```diff
@@ -179,17 +179,14 @@ You can either mount the battery between the MCU and holder PCB, or tape it to t
...
 
-The main branch builds firmware for the colemak dh key layout. The layouts/qwerty branch builds firmware for the qwerty layout. Make sure you pick the correct branch for your needs.
+To add support for the PMW3610 sensor, [inorichi's driver](https://github.com/inorichi/zmk-pmw3610-driver?tab=readme-ov-file) is included in the firmware, along with 
+[Petejohanson's work](https://github.com/petejohanson/zmk/blob/feat/pointers-move-scroll/docs/docs/behaviors/mouse-emulation.md) to allow mouse keys.
 
-![branch names](../assets/pics/bluetooth/branch_names.png)
+The firmware is heavily influenced by Miryoku and homerow mods that use bilateral combinations to make typing as efficient and ergonomic as possible. If you'd like to see the key layout, a layer map is available on the repo's [README](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware).
```
Comment (burkfers) [inline help/bluetooth.md]:
```suggestion
The firmware is heavily influenced by Miryoku and home row mods that use bilateral combinations to make typing as efficient and ergonomic as possible. If you'd like to see the key layout, a layer map is available on the repo's [README](https://github.com/280Zo/charybdis-wireless-mini-zmk-firmware).
```
Spelling

**Review by burkfers — COMMENTED** [review]
Thank you for your submission!

Please also add "nanov" to `.wordlist.txt` as a workaround to our spellchecker also picking up URLs. I cannot add this as a suggestion as github doesn't allow change suggestions on unchanged files.

nb: Spelling only; I cannot comment on the contents.

**Comment by 280Zo** [comment]
Thanks for catching those spelling mistakes! I've accepted the commits, and added "nanov" to the .wordlist.txt.
Let me know if there's anything else.

**Comment by bstiq** [comment]
For some reason my other comment doesn't appear.

What's the reason you deleted those, is it not up to date anymore?


-------------

Note that the official ZMK firmware doesn't support the PMW3610 or mouse movement keys, both of which are used in the firmware above. To get a working firmware I leveraged [the work of inorichi](https://github.com/inorichi) to get the PMW3610 driver, and [the work of petejohanson](https://github.com/petejohanson) for the driver to allow pointer movement and scrolling with keys.

Official ZMK support for mouse keys [is being worked on](https://github.com/zmkfirmware/zmk/pull/778), and when it's merged I'll switch back to the official ZMK firmware for the builds.

**Comment by 280Zo** [comment]
Great question. I should have put that in my initial message.
I'm playing around with other drivers (e.g. the dongle and PMW3610 driver from badjeff), and since it's possible the authors/repos would change, I took that section out.
If you think it's better left in until I merge the changes with main, I'm happy to add it back and then submit another PR if/when the drivers change.

**Comment by burkfers** [comment]
It seems to me you still mention inorichi's driver and Pete's mouse emulation branch, just moved further up (L. 184f). Only the link the the mouse emulation PR is removed.

Imho the link to the driver used should stay, both to tell the reader where to go for more information, and to give credit to the authors.
Another PR would be greatly appreciated once you do change to another driver.

Alternatively, you could add these descriptions to a readme in your repo and link it here; That way you have the freedom to change the documentation as you change the code. It's not directly relevant to the setup on the user's side, so not having it directly on the help page is fine.

**Comment by 280Zo** [comment]
Thanks for the feedback!
I've had the attribution and links in my readme since I started using them.  Screenshot for reference:

---

image

---

Having the info there instead of the setup guide makes sense to me, but I'll leave the final decision up to you. If you prefer it in both locations, let me know and I'll gladly add another commit to restore the text.

**Comment by bstiq** [comment]
Thanks for your contribution! 🥳

**Pull request merged**